src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -162,6 +162,8 @@ bool dropTable(QSqlDatabase& database, const QString& tableName) {
 }
 
 // This function is executed in a separate thread other than the main thread
+// The returned list owns the pointers, but we can't use a unique_ptr because
+// the result is passed by a const reference
```

Comment:
passed where? I don't really understand the problem or how unique_ptr interferes.

Reply:
.. when passed inside QFuture;

Reply:
I will improve the comments